rules/operation-builder.js

```diff
@@ -404,21 +404,7 @@ class ToggleOperation extends OperationConfig {
      */
     doToggle() {
         let item = items.getItem(this.itemName);
```

Comment:
Since we only use the reference once, might be cleaner to do
```suggestion
        items.getItem(this.itemName).toggleCommand();
```

Reply:
Done